pkg/framework/aries/framework.go

```diff
@@ -24,6 +24,7 @@ import (
 type Aries struct {
 	transport                 api.TransportProviderFactory
 	storeProvider             storage.Provider
```

Comment:
TODO - will we need to fix the naming (right now we have "storeProvider" and "transientStoreProvider" - a bit inconsistent).